Source/GSSocketStream.h

```diff
@@ -30,7 +30,7 @@
 #import "GSNetwork.h"
 
 typedef	union {
-  struct sockaddr	s;
+  struct sockaddr_storage	s;
   struct sockaddr_in	i4;
 #ifdef	AF_INET6
   struct sockaddr_in	i6;
```

Comment:
I believe here we actually want sockaddr_in6?

Reply:
I agree, but I didn't change that because I only changed what was necessary to get WebDriver and API testing working.